benefit/data/sheets/EventToComplexTypes/EventToComplexTypes-nonprod.json

```diff
@@ -0,0 +1,33 @@
+[
+  {
+    "CaseEventID": "updateListingRequirements",
+    "CaseFieldID": "panelMemberExclusions",
+    "ListElementCode": "arePanelMembersExcluded",
+    "ID": "updateListingRequirementsArePanelMembersExclusions",
+    "DisplayContext": "OPTIONAL",
+    "LiveFrom": "01/02/2023",
+    "FieldDisplayOrder": 1,
+    "PageID": "1.0"
+  },
+  {
+    "CaseEventID": "updateListingRequirements",
+    "CaseFieldID": "panelMemberExclusions",
+    "ListElementCode": "excludedPanelMembers",
+    "ID": "updateListingRequirementsExcludedPanelMembers",
+    "DisplayContext": "OPTIONAL",
```

Comment:
Shouldn't this be READONLY? OPTIONAL will add the text (Optional) to the element, which I don't think is what you want here?

Reply:
So in the update listing requirements, we want to allow the user to edit the list of excluded panel members but we don't want them to edit the panel
Scenario 2 - Existing exclusions

GIVEN that I need to update panel member exclusions

WHEN I select "Update listing requirements" and go to the screen

THEN display a new section "Panel Members Excluded" 

AND display any existing exclusions

AND allow the user to add new exclusions

AND allow the user to remove exclusions for panel members

Reply:
Oh right so the list of (excludded) panel members is actually writable. Ok